docs/guides/src/main/getting-started/getting-started-openshift.adoc

```diff
@@ -55,35 +55,35 @@ oc process -f https://raw.githubusercontent.com/keycloak/keycloak-quickstarts/la
 | oc create -f -
 ----
 
-Once the command above completes you should see a message similar to this:
-
+. Once the command above completes, look for a message similar to this:
++
 [source,bash,subs="attributes+"]
 ----
 service/keycloak created
 route.route.openshift.io/keycloak created
 deploymentconfig.apps.openshift.io/keycloak created.
 ----
++
+At this point, OpenShift will provision a Keycloak pod and related resources. As part of the process, OpenShift will
+try to pull the Keycloak server image. This operation might take some time depending on your network connection.
 
-At this point, OpenShift is going to provision a Keycloak pod and related resources. As part of the process, OpenShift will
-try to pull the Keycloak server image. This might take some time depending on your network connection.
-
-To make sure Keycloak is provisioned, execute the following command:
-
+. To make sure Keycloak is provisioned, execute the following command:
++
 [source,bash,subs="attributes+"]
 ----
 oc get pods
 ----
 
-After a while you will see a message similar to this when the pod is ready:
-
+. After a while, look for a message similar to the following; it indicates the pod is ready:
++
 [source,bash,subs="attributes+"]
 ----
 NAME                READY     STATUS      RESTARTS   AGE
 keycloak-1-deploy   0/1       Completed   0          1h
 keycloak-1-l9kdx    1/1       Running     0          1h
 ----
 
-Once the server is provisioned, run the following command to find out the URLs of Keycloak:
+. Once the server is provisioned, enter the following command to find out the Keycloak URLs:
 
 [source,bash,subs="attributes+"]
```

Comment:
The bash script should relate to the enumeration item, therefore add a "+" to keep them together.

```suggestion
. Once the server is provisioned, enter the following command to find out the Keycloak URLs:
+
[source,bash,subs="attributes+"]
```